### ISSUE TITLE
'Spell' object has no attribute 'race'

### ISSUE THREAD

**Issue by Ragowit** [jleclanche/fireplace#38]
Deck 1: ['EX1_508', 'CS2_226', 'CS2_142', 'EX1_011', 'CS1_042', 'EX1_011', 'CS2_197', 'EX1_399', 'CS2_173', 'CS2_023', 'CS1_042', 'CS2_122', 'CS2_120', 'DS1_055', 'CS2_162', 'CS2_187', 'CS2_120', 'CS2_141', 'DS1_055', 'CS2_032', 'CS2_127', 'CS2_029', 'CS2_173', 'EX1_399', 'EX1_066', 'EX1_508', 'CS2_026', 'CS2_122', 'CS2_171', 'CS2_187']

Deck 2: ['EX1_009', 'CS2_226', 'GVG_064', 'EX1_556', 'CS2_118', 'EX1_586', 'EX1_572', 'NEW1_016', 'EX1_507', 'GVG_111', 'NEW1_040', 'GVG_092', 'EX1_128', 'EX1_561', 'CS2_120', 'NEW1_025', 'GVG_013', 'EX1_015', 'GVG_098', 'CS2_155', 'NEW1_029', 'CS2_181', 'GVG_112', 'GVG_013', 'GVG_107', 'CS2_196', 'EX1_021', 'EX1_043', 'CS2_189', 'GVG_093']

```
Traceback (most recent call last):
  File "C:\Users\Ragowit\Documents\GitHub\fireplace\mcts\uct.py", line 527, in UCTPlayGame
    m = UCT(rootstate = state, seconds = 10, verbose = False)
  File "C:\Users\Ragowit\Documents\GitHub\fireplace\mcts\uct.py", line 497, in UCT
    state.DoMove(random.choice(state.GetMoves()))
  File "C:\Users\Ragowit\Documents\GitHub\fireplace\mcts\uct.py", line 145, in DoMove
    card.play()
  File "C:\Users\Ragowit\Documents\GitHub\fireplace\fireplace\card.py", line 277, in play
    self.controller.play(self, target, choose)
  File "C:\Users\Ragowit\Documents\GitHub\fireplace\fireplace\player.py", line 212, in play
    self.game.action(PowSubType.PLAY, self, card, target, choose)
  File "C:\Users\Ragowit\Documents\GitHub\fireplace\fireplace\game.py", line 54, in action
    args[0]._play(*args[1:])
  File "C:\Users\Ragowit\Documents\GitHub\fireplace\fireplace\player.py", line 245, in _play
    card.action()
  File "C:\Users\Ragowit\Documents\GitHub\fireplace\fireplace\card.py", line 219, in action
    func(self, **kwargs)
  File "C:\Users\Ragowit\Documents\GitHub\fireplace\fireplace\cards\classic\neutral_epic.py", line 39, in action
    pirates = self.controller.deck.filter(race=Race.PIRATE)
  File "C:\Users\Ragowit\Documents\GitHub\fireplace\fireplace\utils.py", line 35, in filter
    return self.__class__(e for k, v in kwargs.items() for e in self if getattr(e, k) == v)
  File "C:\Users\Ragowit\Documents\GitHub\fireplace\fireplace\deck.py", line 15, in __init__
    super().__init__(cards or [])
  File "C:\Users\Ragowit\Documents\GitHub\fireplace\fireplace\utils.py", line 35, in <genexpr>
    return self.__class__(e for k, v in kwargs.items() for e in self if getattr(e, k) == v)
AttributeError: 'Spell' object has no attribute 'race'
```

```
INFO:root:two begins turn 12
INFO:root:two is now at 6 mana crystals
DEBUG:root:<Minion ('Gnomish Experimenter')> moves from <Zone.DECK: 2> to <Zone.HAND: 3>
INFO:root:two draws <Minion ('Gnomish Experimenter')>
INFO:root:two plays <Minion ('Enhance-o Mechano')> from their hand
DEBUG:root:two summons <Minion ('Enhance-o Mechano')>
INFO:root:Summoning <Minion ('Enhance-o Mechano')>
DEBUG:root:<Minion ('Enhance-o Mechano')> moves from <Zone.HAND: 3> to <Zone.PLAY: 1>
INFO:root:<Minion ('Harvest Golem')> attacks <Hero ('Jaina Proudmoore')>
INFO:root:<Minion ('Harvest Golem')> hits <Hero ('Jaina Proudmoore')> for 2
INFO:root:<Hero ('Jaina Proudmoore')> damaged for 2 health
INFO:root:two plays <Minion ("Captain's Parrot")> from their hand
DEBUG:root:two summons <Minion ("Captain's Parrot")>
INFO:root:Summoning <Minion ("Captain's Parrot")>
DEBUG:root:<Minion ("Captain's Parrot")> moves from <Zone.HAND: 3> to <Zone.PLAY: 1>
INFO:root:Activating <Minion ("Captain's Parrot")> action targeting None
```

**Comment by jleclanche** [comment]
This should be fixed.

**Comment by jleclanche** [comment]
Definitely fixed in fact, by 24204b1336165893b29fd4eb6e2607a64c4bd330
